tests/operator/operator.go

```diff
@@ -2407,20 +2408,56 @@ spec:
 			}
 		})
 
+		// Best effort to speed up this test. The problem is that cdi changes triggers shutdown
+		// of virt-api. The Pod is then restarted and the shutdown is counted towards failures.
+		// This triggeres a backoff which is at max 5 minutes.
+		deleteApiPods := func() {
+			pods, err := virtClient.CoreV1().
+				Pods(flags.KubeVirtInstallNamespace).
+				List(context.Background(), metav1.ListOptions{
+					LabelSelector: fmt.Sprintf("%s=%s", v1.AppLabel, "virt-api"),
+				})
+			Expect(err).ToNot(HaveOccurred())
+			for _, pod := range pods.Items {
+				for _, status := range pod.Status.ContainerStatuses {
+					if status.RestartCount != 0 {
+						err := virtClient.CoreV1().Pods(flags.KubeVirtInstallNamespace).
+							Delete(context.TODO(), pod.Name, metav1.DeleteOptions{})
+						Expect(err).To(Succeed())
+						break
+					}
+				}
+			}
+		}
+
 		It("[test_id:3153][QUARANTINE] Ensure infra can handle dynamically detecting DataVolume Support", decorators.Quarantine, func() {
```

Comment:
Is it worth removing the quarantine label here so that we get a couple of CI runs?